tests_new/integration_tests/conftest.py

```diff
@@ -1,108 +1,134 @@
-import json
+import logging
 import os
+import sys
 from dataclasses import dataclass
-from typing import List
 
 import pytest
+from dataclasses_json import dataclass_json
 
 from tests_new.integration_tests.client import Client
 
+logging.basicConfig(level=logging.INFO, stream=sys.stdout)
 
-## Define user and groups fixtures - We assume they pre-exist in the AWS account.
+pytest_plugins = [
+    'integration_tests.core.organizations.global_conftest',
+    'integration_tests.core.environment.global_conftest',
+]
+
+
+@dataclass_json
 @dataclass
 class User:
     username: str
     password: str
 
-    @staticmethod
-    def from_userdata(userdata, username):
-        return User(username, userdata[username]['password'])
+
+@dataclass_json
+@dataclass
+class Env:
+    accountId: str
+    region: str
+
+
+@dataclass_json
+@dataclass
+class TestData:
+    users: dict[str, User]
+    envs: dict[str, Env]
+
+
+@pytest.fixture(scope='session', autouse=True)
+def testdata() -> TestData:
+    try:
+        return TestData.from_json(open('testdata.json').read())
```

Comment:
This means we can run the tests locally from a testdata.json file right?

Reply:
yes, I introduced it because it's tricky to pass this entire json as an env variable locally.

Reply:
in which path does the file need to be created?

Reply:
current working directory